docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst

```diff
@@ -0,0 +1,272 @@
+.. _Add a Storage Location:
+
+Add a Storage Location
+======================
+
+To add a storage location:
+
+1. Click the **Storage Locations** item in the sidebar to open the
+   **Cloud Locations** window:
+
+   .. image:: ../../Graphics/Orbit_Storage_Locations.png
```

Comment:
A little unrelated but I noticed the copyright in this image was 2018, just wanted to point that out in case it comes up...

Reply:
There shouldn't be Orbit graphics at all by the time I'm done.

Reply:
Okay... the deal is, this ticket was about one thing: add a bucket. This section (location management) I imported from Orbit because links to it from bucket management were broken, and I didn't want to remove the links to make the build work, then forget they had been there. 

Given the problems I'm having with version control, I think I might need to make a monolithic change, rather than going feature by feature...